### PR TITLE
Added in Students section

### PR DESCRIPTION
**Pull request by gautamjajoo**
I have added my details and uploaded my image on the website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,5 +1,3 @@
----
```
Comment (abishekvashok) [inline index.html:1]: Don't remove these lines @gautamjajoo they are needed for jekyll, learn more about jekyll and gh-pages see here: https://help.github.com/categories/github-pages-basics/

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -157,62 +155,47 @@
...
                                        target="_blank">
+                                       <img src="images/icons/twitter.png"
+                                alt="gautam.jpg" style="width:50px; height:50px; padding-bottom:5;">
                                         <i class="fa fa-twitter"></i>
                                     </a>
-                                    {% endif %}
-                                    {% if student.blog %}
-                                    <a href="//{{student.blog}}"
-                                       target="_blank">
-                                        <i class="fa fa-code"></i>
```
Comment (abishekvashok) [inline index.html:196]: You are adding the files to index.html, you should not do that, you only need to include your info to the students file and add your photo to the /images/students section, remember an image has to be 240x240 pixels [w x h]

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by abishekvashok** [comment]
@gautamjajoo ping me on slack if you want more help always there to help you out :)

**Comment by Princu7** [comment]
@gautamjajoo Please resolve the merge conflicts. You can learn more about them in the contribution guide of this project. If you need more assistance, you can also ask us here on ping us on the slack or the gitter channel.
Cheers!

**Comment by abishekvashok** [comment]
You again got it wrong, there is no need to touch the index.html file bro just add your data to the _data/students.yml page and add your image that's. :) Try again... , Happy coding :)

**Review by abishekvashok — APPROVED** [review]

**Comment by abishekvashok** [comment]
@gautamjajoo nice work now :) you know squashing or should I squash and merge for you.

**Pull request merged**